tests/lib/tools/suite/fs-state/task.yaml

```diff
@@ -0,0 +1,45 @@
+summary: smoke test for the fs test tool
+execute: |
+    # Check help
+    "$TESTSTOOLS"/fs-state -h | MATCH "usage: fs-state wait-for-file"
+    "$TESTSTOOLS"/fs-state --help | MATCH "usage: fs-state wait-for-file"
+
+    # Check non existing directory
+    "$TESTSTOOLS"/fs-state ensure-dir-exists "$PWD"/mytestdir
```

Comment:
Is `$PWD` relevant here? I would not use it as it makes the test more complicated than it probably needs to be.

Reply:
$PWD is not needed, I'll remove it